spec/team_spec.rb

```diff
@@ -1,8 +1,46 @@
 require 'spec_helper'
 
-team_path = './data/teams.csv'
```

Comment:
This looks like it got pushed to the bottom, you should probably move it back up.